psx/.cargo/config.toml

```diff
@@ -1,2 +1,10 @@
+[build]
+target = ["mipsel-sony-psx"]
+
 [target.mipsel-sony-psx]
 runner = "mednafen"
+rustflags = ["-Clink-arg=-Tpsexe.ld", "-Clink-arg=--oformat=binary"]
```

Comment:
Is this avoidable? I think it breaks `examples/bios` and `cargo-psx` should add these args anyway.

Reply:
It may or may not break CI, but I'm leaning towards just removing them rather than commenting them out anyway. In case they ever change that avoids things from getting out of sync and possibly misleading anyone.